### PR TITLE
Numerous corrections and updates.

### PR DESCRIPTION
**Pull request by cjschroder**
Made all envvars uppercase. A Tiller service is required. Reviewed and checked every step.

### CODE REVIEW

**Comment by cjschroder** [comment]
@troytop suggested creating the Tiller service with a yaml file, which is a splendid idea. I'll do that after this PR is merged.

File: xml/cap_depl_azure.xml [hunk 1]
```diff
@@ -108,21 +106,32 @@
...
+ 
  <sect1 xml:id="sec.cap.create-aks-instance">
   <title>Create Resource Group and AKS Instance</title>
 
   <para>
    Now you can create a new Azure resource group and AKS instance. Set the
    required variables as environment variables, which helps to speed up the
-   setup, and to reduce errors.
+   setup, and to reduce errors. Verify your environment variables at any time
+   with <command>echo $<replaceable>varname</replaceable></command>, for
```
Comment (btat) [inline xml/cap_depl_azure.xml]: Uppercase `varname`

Comment (cjschroder) [reply xml/cap_depl_azure.xml]: fixxed

File: xml/cap_depl_azure.xml [hunk 2]
```diff
@@ -450,13 +470,41 @@ Private IPs:\t\t\"$(az network nic list --resource-group $MCRGNAME -o json | jq
...
+    <para>
+        You must create a &tiller; service in order to give it sufficient
+        permissions to make changes on your cluster. Run these commands
+        to create your &tiller; service:
+    </para>
+    <screen>&prompt.user;kubectl -n kube-system create serviceaccount tiller
+ serviceaccount "tiller" created
+
+&prompt.user;kubectl create clusterrolebinding tiller \
+--clusterrole cluster-admin \
```
Comment (HartS) [inline xml/cap_depl_azure.xml]: While `cluster-admin` is a pre-defined role in CaaSP, it either needs to be created manually on AKS or another role needs to be used; for QA clusters our AKS deployment script creates this [here](https://github.com/SUSE/cf-ci/blob/f79a83410635cfb55401d1917cbfb2cc378d2bed/qa-tools/deploy-aks.sh#L127), using [this definition](https://github.com/SUSE/cf-ci/blob/f79a83410635cfb55401d1917cbfb2cc378d2bed/qa-tools/cluster-admin.yaml). Only the ClusterRole section of that needs to be deployed for the `kubectl create clusterrolebinding ...` instruction to work.

**Pull request merged**